<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Object List Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelObjectTypes">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Object Types&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelEditArea">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Edit Area&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollAreaForTypes">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>68</width>
         <height>535</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollAreaForEdit">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>394</width>
         <height>535</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonCreateType">
      <property name="text">
       <string>create type</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_List"/>
    <addaction name="actionOpen_List"/>
    <addaction name="separator"/>
    <addaction name="actionSave_List"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="myqrcfile.qrc">
     <normaloff>:/menuIcons/icon-undo-48.png</normaloff>:/menuIcons/icon-undo-48.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="myqrcfile.qrc">
     <normaloff>:/menuIcons/icon-redo-48.png</normaloff>:/menuIcons/icon-redo-48.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionNew_List">
   <property name="icon">
    <iconset resource="myqrcfile.qrc">
     <normaloff>:/menuIcons/icon-add-file-48.png</normaloff>:/menuIcons/icon-add-file-48.png</iconset>
   </property>
   <property name="text">
    <string>New List...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_List">
   <property name="icon">
    <iconset resource="myqrcfile.qrc">
     <normaloff>:/menuIcons/icon-open-48.png</normaloff>:/menuIcons/icon-open-48.png</iconset>
   </property>
   <property name="text">
    <string>Open List...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_List">
   <property name="icon">
    <iconset resource="myqrcfile.qrc">
     <normaloff>:/menuIcons/icon-save-48.png</normaloff>:/menuIcons/icon-save-48.png</iconset>
   </property>
   <property name="text">
    <string>Save List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myqrcfile.qrc"/>
 </resources>
 <connections/>
</ui>
